src/main/java/Duke.java

```diff
@@ -1,10 +1,87 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        Scanner input = new Scanner(System.in);
+        int count = 0; //total number of tasks in the list - 1
+        Task[] Tasks = new Task[100];
+        Greet();
```

Comment:
Hello! Something small I've noticed is that the method Greet() begins with a capital G. Perhaps you could follow the camelCase standard and replace it with greet() instead.